app/main.py

```diff
@@ -1 +1,24 @@
-# write your code here
+from app.players.dwarves.dwarf_warrior import DwarfWarrior
+from app.players.dwarves.dwarf_blacksmith import DwarfBlacksmith
+from app.players.elves.druid import Druid
+from app.players.elves.elf_ranger import ElfRanger
+from app.players.elves.elf import Elf
+from app.players.dwarves.dwarf import Dwarf
+from app.players.player import Player
+
+
+def calculate_team_total_rating(players: list[Player]) -> int:
+    sum_raiting = 0
```

Comment:
There is a typo in the variable name `sum_raiting`. It should be `sum_rating` to correctly reflect the purpose of the variable and maintain consistency with standard spelling.